fail2ban/client/configparserinc.py

```diff
@@ -200,6 +200,18 @@ def get_defaults(self):
 	def get_sections(self):
 		return self._sections
 
+	def options(self, section, onlyOwn=False):
+		"""Return a list of option names for the given section name."""
```

Comment:
I believe that by the function that "Return a list of option names for the given section name" the parameter onlyOwn is enough obvious and speak for itself...